resources/RegexGrammar.pp

```diff
@@ -42,11 +42,13 @@
 //
 
 // Character classes.
+%token  negative_class_fc_       \[\^(?=\])         -> class_fc // same as "negative_class_" but followed by "class:_class"
+%token  class_fc_                \[(?=\])           -> class_fc // same as "class_"          but followed by "class:_class"
```

Comment:
Maybe this would be more human readable? :)

```suggestion
%token  negative_class_fc_       \[\^(?=\])         -> class_fc // same as "negative_class_" but for when the class starts with "]"
%token  class_fc_                \[(?=\])           -> class_fc // same as "class_" but for when the class starts with "]"
```

Reply:
please check the latest changes, comments are not allowed here